skada/_reweight.py

```diff
@@ -669,3 +674,223 @@ def KLIEP(
         ),
         base_estimator,
     )
+
+
+class MMDTarSReweightAdapter(BaseAdapter):
+    """MMDTarSReweight adapter.
+
+    The idea of MMDTarSReweight is to find an importance estimate \beta(y) such that
+    the Maximum Mean Discrepancy (MMD) divergence between the source input density
+    p_source(x) to its estimate p_target(x) is minimized under the assumption
+    of equal conditional distributions p(x|y) for both source and target domains.
+
```

Comment:
Add discussion here (warning block) about how the adapter works and previsdes reweighting for source samples with nearest neighbor else does nothing